### DISCUSSION TITLE
Digital/2024/lab5/tasks/

### DISCUSSION THREAD

**Discussion by giscus** [wintermelon008/USTC-DigitalLab-Discussion#42] [Announcements]
# Digital/2024/lab5/tasks/

https://soc.ustc.edu.cn/Digital/2024/lab5/tasks/

**Comment by EthanCu1** [comment]
请问选做这道题，是把数据存储下来然后并转串，每一位控制状态机跳转这样去做吗，没有太理解题目中的“数据结构知识”是指什么，希望有dl讲解
